.github/workflows/ci-delta.yml

```diff
@@ -36,7 +36,10 @@ jobs:
       - name: StaticAnalysis
         run: sbt -Dsbt.color=always -Dsbt.supershell=false "project delta" clean scalafmtCheck test:scalafmtCheck scalafmtSbtCheck scapegoat
       - name: Tests
-        run: sbt -Dsbt.color=always -Dsbt.supershell=false "project delta" clean coverage test coverageReport coverageAggregate
+        run: |
+          echo "127.0.0.1 bucket.my-domain.com" | sudo tee -a /etc/hosts
```

Comment:
It will add an entry in /etc/hosts everytime

Reply:
Yes and that's fine